mage_ai/frontend/interfaces/BlockType.ts

```diff
@@ -100,6 +100,12 @@ export const BLOCK_TYPE_NAME_MAPPING = {
   [BlockTypeEnum.TRANSFORMER]: 'Transformer',
 };
 
+export const BLOCK_TYPE_PERMANENT = [
```

Comment:
Name can be improved. Sounds like a singular item, perhaps `CONVERTIBLE_BLOCK_TYPES` works better.

Reply:
went with `BLOCK_TYPE_CONVERTIBLE` to match the naming scheme in data cleaning tool (`COLUMN_TYPE_CATEGORICAL`, etc)